### PR TITLE
[WIP] re #202 #203 Integration Action Select & Configure

### PR DESCRIPTION
**Pull request by kahboom**
### Overview
**NOTE:** Still working on this; please don't merge.

**Changes:**
- Worked on Select an Action, replacing the toolbar for that view w/ functional search bar.
- Cleaned up a bit, styled to better reflect designs.
- Edit Integration field in sidebar no longer looks like a field, per design.

**To Do:**
- Going to do the same things for the Configure Action page.
- Need to update the sidebar to display in progress for both pages as well.

### Screenshots
screenshot 2017-03-09 13 28 16

screenshot 2017-03-09 13 31 54

### CODE REVIEW

File: src/app/connections/list-toolbar/list-toolbar.component.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
           </button>
       </div>
       <div class="toolbar-pf-action-right">
-        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['create']">Create
+        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['/connections/create']">Create
```
Comment (gashcrumb) [inline src/app/connections/list-toolbar/list-toolbar.component.html]: You have to do `['connections', 'create']` here, I don't think this works.

Comment (kahboom) [inline src/app/connections/list-toolbar/list-toolbar.component.html]: Yeah that doesn't work but I didn't want to be rude and remove it lol. I'll update that now. Grabbing a drink and then pushing up the changes for the Choose Connections and Configure Actions pages (search bar) and styling. Don't merge yet, please. Almost done!

Comment (kahboom) [inline src/app/connections/list-toolbar/list-toolbar.component.html]: Oh wait, I thought you meant regarding the `showCreate` var.. the routerLink with `/connections/create` seems to be working fine for me.

Comment (kahboom) [inline src/app/connections/list-toolbar/list-toolbar.component.html]: However I don't mind changing it, just wanted to let you know it seems to be working fine. Let me know if you'd like for me to change it.

Comment (gashcrumb) [inline src/app/connections/list-toolbar/list-toolbar.component.html]: ah, k if it works for you then leave it :-)

**Comment by kahboom** [comment]
@gashcrumb - Minor things driving me crazy that I forget (see the search bar on the second screenshot):

screenshot 2017-03-09 16 36 33

screenshot 2017-03-09 16 36 43

**Comment by kahboom** [comment]
Fixed!

screenshot 2017-03-09 16 43 38

Will just need to work on Configure Actions, Sidebar, and some other minor things now (for next PR). :)

**Pull request merged**